includes/utils.php

```diff
@@ -351,6 +347,7 @@ function prepare_meta( $post_id ) {
 				$meta_value = maybe_unserialize( $meta_value );
 				/**
 				 * Filter whether to sync meta.
+				 *
```

Comment:
This is PHPCS - "There must be exactly one blank line before the tags in a doc comment"